audformat/core/database.py

```diff
@@ -289,6 +290,82 @@ def drop_tables(
         for table_id in table_ids:
             self.tables.pop(table_id)
 
+    def files_duration(
+            self,
+            files: typing.Union[str, typing.Sequence[str]],
+            *,
+            root: str = None,
+    ) -> pd.Series:
+        r"""Duration of files in the database.
+
+        Use ``db.files_duration(db.files).sum()``
+        to get the total duration of all files in a database.
+        Or ``db.files_duration(db[table_id].files).sum()``
+        to get the total duration of all files assigned to a table.
+
+        .. note:: Durations are cached,
+            i.e. changing the files on disk after calling
+            this function can lead to wrong results.
+            The cache is cleared when the
+            database is reloaded from disk.
+
+        Args:
+            files: file names
+            root: root directory under which the files are stored.
+                Provide if file names are relative and
+                database was not saved or loaded from disk
```

Comment:
Maybe we should add here, what the default behavior is, e.g.

```suggestion
                database was not saved or loaded from disk.
                If ``None`` :attr:`audformat.Database.root` is used
```

Reply:
Good idea.